drp/README.md

```diff
@@ -0,0 +1,7 @@
+# Dummyrepository (drp)
+## Description: dummyrepository.org is a demonstrator designed to show the possibilities for sharing and publishing the capabilities and metadata of (data) repositories. The general goal is to improve the FAIRness of (data) repositories. To show this, the demonstrator needs a few persistent identifiers.
+
+This [w3id](https://w3id.org/dummyrepository) provides a persistent URI namespace for the [dummyrepository demontrator](https://dummyrepository.org) .
```

Comment:
Was that link supposed to be `drp`?

Reply:
if you wish to reserve three letter ids for larger projects, it would be OK for me to withdraw and change. Just let me know.

Reply:
I was saying you linked to "https://w3id.org/dummyrepository" but this PR is for "https://w3id.org/drp"